hack/rules-and-dashboards/sync_rules.py

```diff
@@ -166,6 +166,10 @@ def cluster_label_var(mo):
 
 
 replacement_map = {
+    "( |\n|\r)+\\) > 0.8": {
```

Comment:
I prefer not to have such hack for rules belong to victoriametrics, but fix the upstream if it's not expected and avoid inconsistencies between both sources.

Reply:
label_value is not supported by promql, but I've opened another PR, which uses metrics, that reports a value of this flag